CityPuzzle/Game Room/Join GameRoom/SeeEnteredRooms.xaml.cs

```diff
@@ -112,7 +112,11 @@ async void Handle_ItemTapped(object sender, ItemTappedEventArgs e)
         async void SelectMsg(Room selectedRoom)
```

Comment:
All page specific methods should be private and if they dont have await in them there is no need for them to be async